myia/prim/py_implementations.py

```diff
@@ -585,3 +585,27 @@ def _list_reduce_vm(vm, fn, lst, dflt):
     def fn_(a, b):
         return vm.call(fn, [a, b])
     return list_reduce(fn_, lst, dflt)
+
+
+@register(primops.embed)
+def embed(node):
+    """Implement `embed`."""
+    raise NotImplementedError()
```

Comment:
This should have an implementation in the debug vm if it can't have one here.

Reply:
OK, I'll do that.